File: python/7 kyu/134.py
# https://www.codewars.com/kata/52f3a8e1f85fadcdf7001e31/train/python

# Passed

def factorial_division(n, d):
    result = 1
    
    # 5! / 3! = 5 * 4 * 3!/3! = 5 * 4
    for i in range(n, d, -1):
        result *= i

    return result

output = factorial_division(6, 3)
print(output)

File: nodejs/7 kyu/134.js
// https://www.codewars.com/kata/5a58d889880385c2f40000aa/train/javascript

// Passed

function automorphic(n) {
  let squaredNumber = n ** 2;
  if (String(squaredNumber).endsWith(String(n))) {
    return "Automorphic";
  }

  return "Not!!";
}

const output = automorphic(25);
console.log(output);